### ISSUE TITLE
Supports `glob` syntax

### ISSUE THREAD

**Issue by hantrungkien** [yusriltakeuchi/assets_cleaner#3]
I mean

```
  exclude-file:
    - /assets/env/**
    - /assets/sounds/**
    - /assets/test/**/**.mp3
```

**Comment by yusriltakeuchi** [comment]
Working on it

**Comment by yusriltakeuchi** [comment]
This feature is released in version 0.1.3+9.  @hantrungkien 
Try the new version here: https://pub.dev/packages/assets_cleaner
